recipe/meta.yaml

```diff
@@ -10,8 +10,8 @@ source:
   sha256: 35a30d83791329429bc04fe44183ce4ab512fcf6968070a7bfba42fc5a0552a9
 
 build:
-  number: 3
-  skip: True  # [win]
+  number: 4
+  skip: True  # [win or py>311]
```

Comment:
Can we pop a comment on top with why py3.11 or above is now skipped so this can be revisited in an update?